mdoc/Mono.Documentation/monodocer.cs

```diff
@@ -408,10 +411,10 @@ private AssemblyDefinition LoadAssembly (string name)
 		AssemblyDefinition assembly = null;
 		try {
 			assembly = AssemblyDefinition.ReadAssembly (name, new ReaderParameters { AssemblyResolver = assemblyResolver });
-		} catch (System.IO.FileNotFoundException) { }
-
-		if (assembly == null)
-			throw new InvalidOperationException("Assembly " + name + " not found.");
+		} 
+		catch (Exception ex) {
+			Warning ($"Error loading assembly '{name}': {ex.Message}");
```

Comment:
If it's a *warning*, then you shouldn't say "Error"; that can make things confusing. I would suggest:

    Warning ($"Unable to load assembly `{name}`: {ex.Message}");